README.md

```diff
@@ -1,114 +1,85 @@
+- `piwikConfig`: An object describing how to contact a [Piwik analytics](https://piwik.org) instance to which usage stats will be sent. Contains the following subproperties:
+    - `url`: The base URL at which the Piwik instance can be contacted. For example: `https://stats.data.gouv.fr`.
+    - `siteId`: The ID of the site to track in this Piwik instance.
+    - `trackErrors`: _A Boolean value._
+- `useCommitReferenceFromApi`: _A Boolean value._
+- `winstonConfig`: A configuration object to pass to the [`winston` logger](https://github.com/winstonjs/winston/tree/2.x#instantiating-your-own-logger) (i.e. what is passed to a `new winston.Logger`).
 
-### Install the necessary helpers
+Some other elements can be configured through environment variables passed to both `npm build` and `npm start`:
 
-Install [Selenium standalone server](http://www.seleniumhq.org/download/).
+- `API_URL`: overrides the `apiBaseUrl` config property.
+- `BASENAME` defines the path at which the Legislation Explorer is served. Defaults to `/`. For example: `BASENAME=/legislation` to serve on `https://fr.openfisca.org/legislation`.
+- `COUNTRY_PRODUCTION_CONFIG`: defines the name of the configuration file to use in the build stage. This file has to be in the `config` folder and be named `production.$COUNTRY_PRODUCTION_CONFIG.js`. For example: `COUNTRY_PRODUCTION_CONFIG=tunisia npm run build`
+- `NODE_ENV` defines if assets should be served minified or with hot module remplacement. Can be either `development` or `production`. Example: `NODE_ENV=production PORT=2030 node index.js`. Prefer using `npm start`.
+- `HOST`: defines the host on which the app is served. Example: `HOST=0.0.0.0 PORT=2030 node index.js`. Prefer using `npm start`.
+- `PORT` defines the port on which the app is served. Example: `NODE_ENV=production PORT=2030 node index.js`. Prefer using `npm start`.
 
-Install Chrome WebDriver : The `tests/config.js` in this package is pre-configured to use Chrome WebDriver.
-* Under Debian GNU/Linux install it with `apt install chromium-driver` (Chromium is the free software version of Chrome).
-* Under Mac OS X install it with `brew install chromedriver`
 
-### Run the integration tests
+## Run the server
 
-- Terminal window 1 : run Selenium standalone server
+`cd` to the cloned `legislation-explorer` directory, and run the following commands:
 
 ```sh
-java -jar selenium-server-standalone-3.4.0.jar
-18:51:36.494 INFO - Selenium build info: version: '3.3.0', revision: 'b526bd5'
-[...]
-18:51:36.722 INFO - Selenium Server is up and running
+npm run build  # compile the frontend code
+npm start  # start the server
 ```
-- Terminal window 2 : run Chrome WebDriver
 
-```sh
-chromedriver
-Starting ChromeDriver 2.25 (undefined) on port 9515
-Only local connections are allowed.
-```
 
-- Terminal window 3 : start the application development server if not already done (`npm run dev:prod-api`).
+## Improve the app
```

Comment:
I think information about the default behavior (openfisca-france and localhost:2030) would be helpful here.

Reply:
This information is output by the server when it starts. I would prefer to avoid duplicating it in documentation as it is quite likely to be out of date there.